.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
```

Comment:
Was as described in https://github.com/babel/babel-eslint but I deleted it anyway.